hibernate-core/hibernate-core.gradle

```diff
@@ -124,7 +121,9 @@ dependencies {
     testRuntime( libraries.weld )
     testRuntime( libraries.atomikos )
     testRuntime( libraries.atomikos_jta )
-    testRuntime(libraries.wildfly_transaction_client)
+    testRuntime( libraries.wildfly_transaction_client )
+    testRuntime( libraries.junit5_vintage )
+    testRuntime( libraries.junit5_jupiter )
 
```

Comment:
they can be removed, the correct dependencies are already declared in `java-module.gradle`